.env

```diff
@@ -41,8 +41,10 @@ IMGPROXY_USE_S3=false
 
 # api
 API_PORT=4000
+API_PROXY_PORT=4001
+MITMPROXY_WEB_PASSWORD=password
```

Comment:
Unfortunately there is a password required to open the web interface: https://github.com/mitmproxy/mitmproxy/issues/7551

Due to this pull request aims to use it only as a dev dependency this would not be necessary for us.